### ISSUE TITLE
How should we evaluate the MTT / LDH models?

### ISSUE THREAD

**Issue by jessica-ewald** [broadinstitute/2024_09_09_Axiom_OASIS#6]
We've shown that simple XGBoost regression models do a much better job of predicting MTT and LDH values than simple baselines like correlation with cell count. Currently the model is just 5-fold cross validation with random splits. It will be important to construct more challenging scenarios and see how performance holds up. The key question is whether it makes more sense to split the current data in different ways, or if there are substantially different batches in the 50k Axiom set that we should use for validation. 

Relevant comments from Google doc here @AnneCarpenter and @ktitterton:

Anne: "See if Axiom has collected any data on compounds that are NOT in the set you gave us, where you have Cell Painting images + any of the assay readouts already in the paper (e.g. MTT). Or, if you are willing to grab a few hundred compounds and do so. Then Jess can predict MTT outcomes on those compounds based on the images and see how accurate. The challenge will be batch effects but it may work well.

Katherine: "Constructing a validation task from our larger dataset of 50k unique small molecules is a very reasonable and feasible ask. We can use the ridge normalized MTT values which effectively eliminate positional and plate effects, but we do still see batch effects for a minority of compounds (~5%) - some compounds look less potent in the 2nd batch replicate."

Anne: "Actually, Instead of Jess sending you a list of compounds she hopes you have data on, does Axiom already have plates of compounds that have Cell Painting + MTT (or LDH) assays done on them? If so, that will allow her to quickly accomplish validation. If not, then you could send her Cell Painting data for a plate of compounds and then you can run the MTT or LDH assay on those compounds separately."

Jess: "So - I thought that Axiom already has MTT/LDH for the full 50k compound set, and that the 50k set + OASIS compounds were collected pretty much at the same time? If so, I don't necessarily see the benefit of applying the model I've trained to more of their data and then checking the MTT/LDH readouts when I could just do that same thing with the data we already have - I would just create a held out test set, with splits by plate/compound to make it harder.

Unless, as Anne mentioned, there are significantly different batches for the 50k compound set vs. what we have. In that case, it might be sufficient to get a plate's worth of paired cell painting images + assay read-outs, just to see how it generalizes across batch."
